commands/initial/templates/package.json

```diff
@@ -15,7 +15,9 @@
     "prebuild": "rimraf dist out-tsc",
     "start": "webpack-dev-server --open --config ./webpack/webpack.dev.js",
     "tagVersion": "np --no-publish",
-    "test": "node ./tasks/test"
+    "test": "node ./tasks/test",
+    "pree2e": "webdriver-manager update",
+    "e2e": "protractor"
```

Comment:
If i understand correctly you want to run the protractor tests along with the unit tests? I think that would cause a few issues

- Protractor tests are very expensive to run. and would take a lot more time to run then unit tests. 
- When the user clicks save the tests most likely would not have finished executing before the next save
- I think running protractor tests on save would be very annoying for the developer as it would be opening real browsers, basically robbing control from the user until the tests completed.
- Keeping the tests in separate scripts is the recommended approach of the angular-cli